site/components/product/Swatch/Swatch.module.css

```diff
@@ -1,9 +1,9 @@
 .swatch {
   box-sizing: border-box;
   composes: root from '@components/ui/Button/Button.module.css';
-  @apply h-10 w-10 bg-primary text-primary rounded-full mr-3 inline-flex
+  @apply h-10 w-10 bg-primary text-primary rounded-full inline-flex
   items-center justify-center cursor-pointer transition duration-150 ease-in-out
-  p-0 shadow-none border-accent-3 border box-border select-none;
+  p-0 shadow-none border-accent-3 border box-border select-none !mr-0;
```

Comment:
What is `!mr-0`? First time seeing it 👀

Reply:
The ! before any tailwind class is like setting  !important, swatch had a margin-right (that I thought comes from the button that composes, but it was following line. So I removed the margin-right and added a gap to the parent because when the swatch wrapped and fell underneath, it didn't have a margin-top.